### PR TITLE
Using Media Queries

### PR DESCRIPTION
**Pull request by kmcmakin00**
I am planning on using Media Queries that change the layout depending on screen size.

### CODE REVIEW

**Review by yashaswini-slu — CHANGES_REQUESTED** [review]
i can still see the same design for the footer
Screenshot 2024-02-25 at 16 08 25

**Review by yashaswini-slu — CHANGES_REQUESTED** [review]
Screenshot 2024-03-18 at 11 11 55
we can improve this design

**Review by yashaswini-slu — CHANGES_REQUESTED** [review]
we can add tooltip - helper text, user will get an idea of what that icon is for.
Screenshot 2024-03-18 at 11 14 38

**Review by yashaswini-slu — CHANGES_REQUESTED** [review]
Update the branch with main branch

File: public/components/layout.js [hunk 1]
```diff
@@ -8,529 +8,542 @@
...
-                src="https://centerfordigitalhumanities.github.io/media-assets/logos/rcg-logo.png"></a>
-        <a target="_blank" href="http://rerum.io"><img class="brand" alt="Rerum logo"
-                src="https://centerfordigitalhumanities.github.io/blog/assets/images/rerum.jpg"><small>RERUM
-                v1</small></a>
-	    <a target="_blank" href="https://github.com/CenterForDigitalHumanities/geolocator.git"><img width="30" height="30" alt="Git logo" src="./github.svg">
-        <small>&nbsp;&nbsp;Support/Contribute to Geolocator</small></a>
+
+	<a target="_blank"
+		href="https://www.slu.edu/research/faculty-resources/research-computing.php">
+		<small class = "tooltiptext">&copy;2022 Research Computing Group</small></a>
```
Comment (thehabes) [inline public/components/layout.js:77]: &copy;**2024** Research Computing Group

File: public/components/layout.js [hunk 2]
```diff
@@ -8,529 +8,542 @@
...
+
+	<a target="_blank" class="tooltip" href="https://slu.edu">
+		<img class="brand" alt="SLU logo" src="https://www.slu.edu/marcom/tools-downloads/imgs/logo/left-aligned/slu_logoleftaligned_rgb.png">
+		<span class= "tooltiptext">SLU Homepage</span></a>
+
+	<a target="_blank" class = "tooltip" id="slu" href="https://www.slu.edu/arts-and-sciences/ong-center/">
+		<img class="brand" alt="SLU RCG logo" src="https://centerfordigitalhumanities.github.io/media-assets/logos/rcg-logo.png">
+		<span class= "tooltiptext">SLU Research Group</span></a>
```
Comment (thehabes) [inline public/components/layout.js:85]: Just "SLU Research"

File: public/components/layout.js [hunk 3]
```diff
@@ -8,529 +8,542 @@
...
+
+
+	<a target="_blank" class= "tooltip"id = "rerum" href="http://rerum.io">
```
Comment (thehabes) [inline public/components/layout.js:88]: Also not your fault.  Finally got our act together and have https

`href="https://rerum.io"`

File: public/components/layout.js [hunk 4]
```diff
@@ -8,529 +8,542 @@
...
-    constructor(){
-        super()
-        const header = document.createElement('geo-header')
-        const navbar = document.createElement('geo-nav')
-        const footer = document.createElement('geo-footer')
-        this.prepend(header)
-        this.append(navbar)
-        this.append(footer)
-    }
+	constructor(){
```
Comment (thehabes) [inline public/components/layout.js:11]: Try to `undiff` whitespace changes like this if you didn't change the code.  Changing the whitespace can have consequences, some bracket-less languages really care about this and it is a bad habit to form.

File: public/stylesheets/style.css [hunk 5]
```diff
@@ -1,128 +1,216 @@
 body{
-  margin-bottom: 20rem;
+	margin-bottom: 20rem;
 }
 footer{
-  bottom : 0;
-  background-color : #FFF;
-  z-index : 1;
+	bottom : 0;
```
Comment (thehabes) [inline public/stylesheets/style.css:5]: Try to `undiff` whitespace changes like this if you didn't change the code.  Changing the whitespace can have consequences, some bracket-less languages really care about this and it is a bad habit to form.

**Review by thehabes — CHANGES_REQUESTED** [review]
I do see a great deal of improvement in how the header/footer is handle as I move between device emulators, this is much better.  I am impressed that both landscape and portrait mode were taken into account.

I can see the footer icons just did not want to line up for you in some cases.  [Using a "grid" technique](https://jenil.github.io/chota/#grid) may help this be more consistent.  Remember that you have Chota CSS in the background, and using the `row` and `column` classes can do a lot of this as well.  It is OK as-is, just leaving this note.

**Galaxy S20+**
image

**Galaxy S8**
image

I notice a lot of whitespace changes where the tab index has been changed that should be cleaned up.  If you look at your diff, then apply the "hide whitespace changes" settings and see it again, you'll see how much "unchanged" code is affected (it is significant).

image

**Make sure to resolve the conflicts** (I think the whitespace is involved here.)  I have noted some links to update and some cleanup items inline.  Just some cleanup stuff to think about polishing this up before taking it forward.

**Pull request closed without merging**